index.html

```diff
@@ -3858,7 +3858,11 @@ <h4>updated</h4>
 DID document metadata SHOULD include an <code>updated</code> property to
 indicate the timestamp of the last <a href="#update">Update operation</a> for
 the document version which was resolved. The value of the property MUST follow
-the same formatting rules as the <code>created</code> property.
+the same formatting rules as the <code>created</code> property. The <code>updated</code> property
+is omitted if an Update operation has never been performed on the DID document.
+If an <code>updated</code> property exists, it can be the same value as the
+<code>created</code> property when the difference between the two timestamps
+is less than one second.
```

Comment:
Easier to do this now than later... Just whitespace.
```suggestion
the same formatting rules as the <code>created</code> property. The
<code>updated</code> property is omitted if an Update operation has never been
performed on the DID document. If an <code>updated</code> property exists, it
can be the same value as the <code>created</code> property when the difference
between the two timestamps is less than one second.
```

Reply:
Thanks:)